orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -77,15 +77,45 @@ class OperationsController {
 
   @RequestMapping(value = "/orchestrate", method = RequestMethod.POST)
   Map<String, Object> orchestrate(@RequestBody Map pipeline, HttpServletResponse response) {
-    parsePipelineTrigger(executionRepository, buildService, pipeline)
-    Trigger trigger = objectMapper.convertValue(pipeline.trigger, Trigger)
+    if (pipeline.plan ?: false) {
+      parseAndValidatePipeline(pipeline)
+      log.info('not starting pipeline (plan: true): {}', value("pipelineId", pipeline.id))
```

Comment:
I think we generally capitalize the start of these log message

Reply:
Fixed.